README.md

```diff
@@ -0,0 +1,112 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#Setup)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_Microverse] <a name="about-project"></a>
+
+It is a first introduction project in which we have to learn how Review code.
+
+# [Hello_Microverse] is a first microverse project . To master all of the tools and practices.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+I'm using mainly html and css for building this project.
+
+> <details>
+
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.

```

Comment:
```suggestion
> Describe between 1-3 key features of the application.

```
- [ ] You're required to remove all the descriptive section. They are only there to inform you what it is expected of you to do on that particular section. Leaving it after you've written what is supposed to be there does not make your README file look professional. Kindly remove all the descriptive section.